engine-tests/src/test/java/org/terasology/rendering/dag/SetName.java

```diff
@@ -58,4 +49,7 @@ public String toString() {
     public String getName() {
         return name;
     }
+
+    @Override
```

Comment:
If this method took the place of the removed generateTask() method this diff would be slightly easier to browse. No big deal though.